server/routes/index.ts

```diff
@@ -74,6 +78,66 @@ router.get('/health', async (req, res) => {
   res.send('OK').end()
 })
 
+const substackDomainCache = {
+  // uncomment the below for test purpose
+  'localhost:3100': 'polymorpher.substack.com'
```

Comment:
just use polymorpher.localhost:3100 in Chromium-powered browsers, it will still map to localhost:3100 but now you get SLD

Reply:
Done